src/screens/HoursScreen.tsx

```diff
@@ -1,17 +1,80 @@
 import React from 'react';
-import { View, Text, StyleSheet } from 'react-native';
-import baseStyle from '../styles/base';
+import {
+  View,
+  Text,
+  StyleSheet,
+  SafeAreaView,
+  SectionList,
+} from 'react-native';
+import colors, { theme } from '../styles/colors';
+import Constants from 'expo-constants';
+
+const DATA = [
+  {
+    title: 'Request A Walk',
+    data: ['Monday'],
+  },
+  {
+    title: 'Request SeeUs Van',
+    data: [],
+  },
+  {
+    title: 'Request A Jump/Unlock',
+    data: [],
+  },
+];
```

Comment:
No hours data?

Reply:
Smh I deleted it trying to fix commit and forgot to add it back